tests/kernel/queue/src/test_queue_user.c

```diff
@@ -58,9 +57,11 @@ void child_thread_get(void *p1, void *p2, void *p3)
 }
 
 /**
- * @brief Verify queue elements from a user thread
+ * @brief Verify queue elements and cancle wait from a user thread
```

Comment:
"cancel wait"

Reply:
I'm sorry, and I'll be going to pay more attention to spelling, which has been changed.